compose/reference/scale.md

```diff
@@ -13,3 +13,10 @@ Sets the number of containers to run for a service.
 Numbers are specified as arguments in the form `service=num`. For example:
 
     docker-compose scale web=2 worker=3
+
+>**Tip:** Alternatively, in
+[Compose file version 3.x](/compose/compose-file/index.md)
+with Docker Engine 1.13 and above, you can specify
+[`replicas`](/compose/compose-file/index.md#replicas) under
+[`deploy`](/compose/compose-file/index.md#deploy) as part of the service
```

Comment:
I'll leave it up to you, but is "under deploy" here really necessary? It should become obvious as soon as readers click through the `replicas` link.